editor/src/core/shared/multiplayer-hooks.tsx

```diff
@@ -0,0 +1,27 @@
+import { useAtom } from 'jotai'
+import React from 'react'
+import {
+  ActiveRemixSceneAtom,
+  RemixNavigationAtom,
+} from '../../components/canvas/remix/utopia-remix-root-component'
+import type { RemixPresence } from './multiplayer'
+import * as EP from './element-path'
+
+export function useRemixPresence() {
```

Comment:
For my own benefit, could we have a return type on this?

Reply:
https://github.com/concrete-utopia/utopia/pull/4552/commits/16a4204fc8134fd83a31ea2d8bdd89accaaf26d8